CommonJS lambda broken

## Description

When installing NR layer for CommonJS lambda, the CLI overrides handler to `newrelic_lambda_wrapper.handler` instead of `newrelic-lambda-wrapper.handler`

## Steps to Reproduce

run `newrelic-lambda layers install --upgrade` for any NodeJS lambda without the `--esm` flag

## Expected Behaviour

lambda function setup with NR layer, and the handler should be `newrelic-lambda-wrapper.handler`

## Relevant Logs / Console output

lambda execution failed with

```
{
    "errorType": "Runtime.ImportModuleError",
    "errorMessage": "Error: Cannot find module 'newrelic_lambda_wrapper'\nRequire stack:\n- /var/runtime/index.mjs",
    "stack": [
        "Runtime.ImportModuleError: Error: Cannot find module 'newrelic_lambda_wrapper'",
        "Require stack:",
        "- /var/runtime/index.mjs",
        "    at _loadUserApp (file:///var/runtime/index.mjs:1087:17)",
        "    at async UserFunction.js.module.exports.load (file:///var/runtime/index.mjs:1119:21)",
        "    at async start (file:///var/runtime/index.mjs:1282:23)",
        "    at async file:///var/runtime/index.mjs:1288:1"
    ]
}
```

## Additional context

Bug introduced in the following feature:
https://github.com/newrelic/newrelic-lambda-cli/pull/314/files#diff-d14719dcdcf446bd00c4b7977b0acee473aa1e5dbce20b230ccca5e856e3c9cfR107

--- a/newrelic_lambda_cli/layers.py
+++ b/newrelic_lambda_cli/layers.py
@@ -104,7 +104,7 @@ def _add_new_relic(input, config, nr_license_key):
         prefix = (
             "/opt/nodejs/node_modules/newrelic-esm-lambda-wrapper/index"
             if input.esm
-            else "newrelic_lambda_wrapper"
+            else "newrelic-lambda-wrapper"
         )
         runtime_handler = prefix + ".handler"
 

--- a/tests/test_layers.py
+++ b/tests/test_layers.py
@@ -333,7 +333,7 @@ def test_add_new_relic_nodejs(aws_credentials, mock_function_config):
     assert (
         update_kwargs_std["FunctionName"] == config_std["Configuration"]["FunctionArn"]
     )
-    assert update_kwargs_std["Handler"] == "newrelic_lambda_wrapper.handler"
+    assert update_kwargs_std["Handler"] == "newrelic-lambda-wrapper.handler"
     assert (
         update_kwargs_std["Environment"]["Variables"]["NEW_RELIC_LAMBDA_HANDLER"]
         == original_std_handler

